test/testcafe/spec/Redirect_spec.js

```diff
@@ -0,0 +1,101 @@
+import { RequestMock, RequestLogger } from 'testcafe';
+
+import ChallengeEmailPageObject from '../framework/page-objects/ChallengeEmailPageObject';
+import SuccessPageObject from '../framework/page-objects/SuccessPageObject';
+import { oktaDashboardContent } from '../framework/shared';
+
+import emailVerificationWithPolling from '../../../playground/mocks/data/idp/idx/authenticator-verification-email-polling-short';
+import successRedirectRemediation from '../../../playground/mocks/data/idp/idx/success-redirect-remediation';
+
+const emailPollSuccessMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(emailVerificationWithPolling)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/poll')
+  .respond(successRedirectRemediation)
+  .onRequestTo(/^http:\/\/localhost:3000\/app\/UserHome.*/)
+  .respond(oktaDashboardContent);
+
+const pollRequestLogger = RequestLogger(/poll/);
+
+fixture('Redirect on Polling');
+
+function getChromeClient(t) {
+  // Get internal Chrome DevTools client
+  const openedBrowsers = t.testRun.browserConnection.provider.plugin.openedBrowsers;
+  const openedBrowser = openedBrowsers[Object.keys(openedBrowsers)[0]];
+  const clients = openedBrowser.browserClient._clients;
+  const client = clients[Object.keys(clients)[0]].client;
+  return client;
+}
+
+async function openNewTab(t, url = '') {
+  // https://chromedevtools.github.io/devtools-protocol/tot/Target/#method-createTarget
```

Comment:
There is no way to open new tab with TestCafe API, only new window. But we can use DevTools